Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithInProcessRenderingBackingStore.mm

```diff
@@ -103,6 +103,7 @@
     }
 
     GraphicsContext& context = m_frontBuffer.imageBuffer->context();
+    GraphicsContextStateSaver outerSaver(context);
```

Comment:
I put it here because I had some vague memory that image drawing also sets state? But it may reset it itself.

You're right that outer is meaningless.